lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1986,6 +1986,12 @@ def cmd_set_help
   # Sets a name to a value in a context aware environment.
   #
   def cmd_set(*args)
```

Comment:
Auxiliary modules have the ability to implement new console commands, which doesn't require modifications to the core command dispatcher. Lets chat about how to port this to in-memory profiles and auxiliary.